exonum-java-binding/core/src/main/java/com/exonum/binding/core/runtime/ServiceRuntime.java

```diff
@@ -386,6 +389,37 @@ public void connectServiceApis(int[] serviceIds, Node node) {
     }
   }
 
+  /**
+   * Get service instance by its name.
+   *
+   * @throws IllegalArgumentException if there is no service with such name
+   */
+  public Service getServiceInstanceByName(String serviceName) {
+    return findService(serviceName)
+        .map(ServiceWrapper::getService)
+        .orElseThrow(() ->
+            new IllegalArgumentException("No service with such name in the Java runtime "
+                + serviceName));
+  }
+
+  /**
+   * Converts an Exonum raw transaction to an executable transaction of given service.
+   *
+   * @param serviceId the id of the service
+   * @param txId the {@linkplain TransactionMessage#getTransactionId() transaction type identifier}
+   *     within the service
+   * @param arguments the {@linkplain TransactionMessage#getPayload() serialized transaction
+   *     arguments}
+   * @return an executable transaction of the service
+   * @throws IllegalArgumentException of there is no service with such name in this runtime, or if
+   *     the transaction is not known to the service, or the arguments are not valid: e.g., cannot
+   *     be deserialized, or do not meet the preconditions
+   */
+  public Transaction convertTransaction(int serviceId, int txId, byte[] arguments) {
```

Comment:
(1) Why does this method return transaction if what testkit needs is transaction
  verification (#verifyTransaction -> void) which throws exception if it is not valid? Or is there are any other uses?
  (2) Why isn't this operation inside ServiceWrapper, which already implements it properly,
  see the first lines of ServiceWrapper#execute?
  (3) Thread-safe.
  (4) Tests (mostly would be appropriate for SW).